### ISSUE TITLE
"invalid character 'ï' looking for beginning of value"

### ISSUE THREAD

**Issue by typeoneboy** [alpkeskin/mosint#43]
When I run `sudo go run mosint <email>`, it gives me this error:

> |invalid character 'ï' looking for beginning of valueinvalid character 'ï' looking for beginning of valueinvalid character 'ï' looking for beginning of valueinvalid character 'ï' looking for beginning of valueinvalid character 'ï' looking f
> Target Email: <email>
> |--> Verified ✔
> |--> Not Disposable ✔
> invalid character 'ï' looking for beginning of valueinvalid character 'ï' looking for beginning of value
> Social Media Results:
> |- Spotify 👀
> |- Twitter 👀
> |- Discord 👀
> |- Adobe 👀
> |- Instagram 👀
> invalid character 'ï' looking for beginning of valueinvalid character 'ï' looking for beginning of valueinvalid character 'ï' looking for beginning of value
> Lookup Results:
> +-----+------------------------------------------------------------------+
> | IP  | [redacted]                                                              |
> | IP  | [redacted]                                                              |
> | NS  | ns3.google.com.                                                  |
> | NS  | ns4.google.com.                                                  |
> | NS  | ns2.google.com.                                                  |
> | NS  | ns1.google.com.                                                  |
> | MX  | gmail-smtp-in.l.google.com.                                      |
> | MX  | alt1.gmail-smtp-in.l.google.com.                                 |
> | MX  | alt2.gmail-smtp-in.l.google.com.                                 |
> | MX  | alt3.gmail-smtp-in.l.google.com.                                 |
> | MX  | alt4.gmail-smtp-in.l.google.com.                                 |
> | TXT | v=spf1                                                           |
> |     | redirect=_spf.google.com                                         |
> | TXT | globalsign-smime-dv=[redacted]       |
> +-----+------------------------------------------------------------------+

this is my mosint-config.json file
`{
 "Breachdirectory": "<API key>",
 "Hunter": "<API key>",
 "Emailrep": "",
 "Intelx": "<API key>",
 "Psbdmp": "True"
}`

I have also noticed it is does not seem to be using any of my config. Any help would be appreciated. I have also noticed it is does not seem to be using any of my config. Cheers

**Comment by RokeJulianLockhart** [comment]
I'm getting

> ```log
> invalid character ':' after top-level valueinvalid character ':' after top-level value
> ```

My config file is

<blockquote>

```JSONC
{
	"Breachdirectory": "",
	"Emailrep": "",
	"Hunter": "",
	"Intelx": "",
	"Psbdmp": ""
}
```

</blockquote>

per https://github.com/alpkeskin/mosint/issues/62#issuecomment-1557416366.

**Comment by alpkeskin** [comment]
Bugs have been fixed in the new version (v3.0.0). Ensure the config file is configured correctly for **Mosint** to work properly.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/alpkeskin/mosint/issues/43#issuecomment-1726544177

@alpkeskin, do you mean that *this* bug has been fixed?
